meta.go

```diff
@@ -73,10 +262,13 @@ func main() {
 				if len(c.Args()) <= 1 {
 					return cli.ShowAppHelp(c)
 				}
-				metaSpace := c.String("meta-space")
-				setMeta(c.Args().Get(0), c.Args().Get(1), metaSpace)
+				err := setMeta(c.Args().Get(0), c.Args().Get(1), metaSpace)
+				if err == nil {
```

Comment:
Fixed.

First, I fixed finalRecover() to call failureExit(). When main() return nil, `defer finalReciver()` called and exit with failure. But it always exit with non-zero even show version...